graylog2-web-interface/src/components/search/MessageFieldDescription.jsx

```diff
@@ -50,6 +50,15 @@ class MessageFieldDescription extends React.Component {
     SearchStore.addSearchTerm(this.props.fieldName, this.props.fieldValue);
   };
 
+  _getFieldValue() {
```

Comment:
Please use an arrow function here, to ensure the method is bound to `this` in any case.